Aptoide/aptoide-client-v8
# [Feature] Wallet Verification

**What does this PR do?**

   This PR corresponds to the implementation of the feature of wallet verification

**Database changed?**

   No

**Where should the reviewer start?**

- [ ] ClaimPromotionsPresenter.java

**How should this be manually tested?**

  Open app and check that the flow for the wallet verification is working correctly.

**What are the relevant tickets?**

  Tickets related to this pull-request: [ASV-1820](https://aptoide.atlassian.net/browse/ASV-1820)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new keys, etc.) 



**Code Review Checklist**
